src/NLog/LogFactory.cs

```diff
@@ -1260,11 +1260,11 @@ private void ConfigFileChanged(object sender, EventArgs args)
         /// <summary>
         /// Logger cache key.
         /// </summary>
-        internal class LoggerCacheKey : IEquatable<LoggerCacheKey>
+        internal struct LoggerCacheKey : IEquatable<LoggerCacheKey>
```

Comment:
What's the idea of the struct? AFAIK it's not in line with https://msdn.microsoft.com/en-us/library/ms229017(v=vs.110).aspx ?